src/cmd/create.go

```diff
@@ -710,18 +710,42 @@ func pullImage(image, release, authFile string) (bool, error) {
 		shouldPullImage = true
 	}
 
+	imageSizeChan := make(chan string)
+
 	if promptForDownload {
 		fmt.Println("Image required to create toolbox container.")
+		// Launch a goroutine to call getImageSizeFromRegistry and send the size to the channel
+		go func() {
+			imageSize, err := getImageSizeFromRegistry(imageFull)
+			if err != nil {
+				logrus.Debugf("Getting image size failed: %s", err)
+				imageSize = ""
+			}
+			imageSizeChan <- imageSize
+		}()
+
+		// Show the download question immediately
+		prompt := fmt.Sprintf("Download %s (...)? [y/N]:", imageFull)
+		fmt.Printf("%s", prompt)
+
+		// Wait until the size is received or a timeout occurs
+		timeout := time.After(5 * time.Second)
```

Comment:
Do we really need a timer?  We could have a timer that prevents against  `skopeo inspect` getting stuck for some reason, but it's probably not that important or necessary.  eg., we don't worry about `podman pull` getting stuck.  If there's some problem with the network, I would expect `skopeo inspect` to eventually time out on its own.

For the vast majority of cases, the user will act as the *timer*.  If `skopeo inspect` is too slow, then the user would probably decide `yes` or `no` before it fetches the size.  That's the thing that should really work.

Reply:
I agree, I better remove the timer.